### ISSUE TITLE
Add the option to display PP as a time instead of a number

### ISSUE THREAD

**Issue by dhouck** [Aktanusa/CookieMonster#625]
As the name indicates, the Payback Period is a time period.  Instead of seeing that eg. a cursor at the start of the game has a PP of 150, it would be nice to see that this was 2m 30s.  This also makes it easier to compare with the times listed in the warnings.

**Comment by dhouck** [comment]
Thanks for adding this.

I do want to defend the request a bit given the notes: it *is* a time period.  It's cost/Δcps.  That's something with units "cookies" divided by something with units "cookies per second".  [cookies]/([cookies]/[time]) = [time][cookies]/[cookies] = [time].

It doesn't translate directly into a time because wrinklers, buffs, debuffs, other bonuses/penalties, other purchases/sales, changing multipliers like garden plants and century eggs, achievements and milk, etc. might all change the actual time involved.  But that's also true of other things displayed as a time, like the "time left" until you're able to purchase something.  If don't have a garden, wrinklers, or century egg, and you leave the game alone, the payback period should in fact be the exact amount of time until the purchase breaks even.

**Comment by DanielNoord** [comment]
Yeah I understand that it is a time unit, perhaps we should description to be more specific. What I mean is that PP also incorporates the time it takes to buy the upgrade. Therefore, it is not only a "breakeven-counter" in the strictest sense. This also means that if other variables changes, the PP can also change. We have had some issues raised about this before so I wanted to make it clear to users that they should not expect to always be able to just sit there and wait for the building to break even.
